openmls/src/framing/mod.rs

```diff
@@ -28,6 +28,7 @@ pub(crate) use sender::*;
 
 // Public
 pub use ciphertext::*;
+pub use message::*;
```

Comment:
You should remove the `pub(crate)` in line 26.

Reply:
I am not sure I follow, are you suggesting making it `pub` rather than `pub(crate)`?

Reply:
You made it `pub` in line 31, which supersedes the `pub(crate)` in line 26. So you can delete line 26.

Reply:
oops... good catch.